portal-web/test/functional/com/liferay/portalweb/macros/headless/headlessadmintaxonomy/TaxonomyVocabularyAPI.macro

```diff
@@ -0,0 +1,136 @@
+definition {
+
+	macro _curlTaxonomyVocabulary {
```

Comment:
All methods with JSONCurlUtil based on this _curlTaxonomyVocabulary() method, in which defined what the api for taxonomy-vocabularies is, as well as a base curl with url, username/password and header etc.